bond_app/bond.py

```diff
@@ -60,23 +60,61 @@ def exchange_authz_code(self, authz_code, redirect_uri, sam_user_id):
                                       jwt_token.username, self.provider_name)
         return jwt_token.issued_at, jwt_token.username
 
-    def generate_access_token(self, sam_user_id):
+
+    def generate_access_token(self, sam_user_id, refresh_token=None):
         """
-        Given a user, lookup their refresh token and use it to generate a new refresh token from their OAuth
+        Given a user, lookup their refresh token (if not provided) and use it to generate a new refresh token (<-should this be "access token"?) from their OAuth
```

Comment:
see inline "comment" -- given my understanding, I think this should read as follows:
> Given a user, lookup their refresh token (if not provided) and use it to generate a new *access token* from their OAuth provider.

Reply:
agreed